docs/development.rst

```diff
@@ -255,7 +329,7 @@ The following examples show to extend the images.
            -e ZOOKEEPER_LOG_CONFIG_URL =http://foo.com/zk1/log4j.properties \
            foo/zookeeper:latest
 
-2. Add More Software
+3. Add More Software
```

Comment:
tip: you can use `#.` instead of actual numbers. when the output is generated, the actual numbers are automatically calculated.